TMIS March (#537)
solid/solidproject.org: _posts/newsletter/next.md

### [Events](https://solidproject.org/events)

**11 March 2021** — It was exciting to see the great turnout at March’s Solid World. We continue to grow in the Solid community as reflected in Solid World’s highest attendance ever. During the event, we got to touch on a lot of great ideas related to the growing market of interoperability. 

> Tim Berners-Lee kicked off the event with shout-outs to last month’s Community Spotlights: Virginia Balseiro for building Women of Solid and [Digita](https://www.digita.ai/) for their [introduction videos](https://www.youtube.com/channel/UCA22hu-0VEHt5tCc7jad74g) as well as their contributions in discovering and reporting bugs.

Reviewer: ```suggestion
Tim Berners-Lee kicked off the event with shout-outs to last month’s Community Spotlights: [Virginia Balseiro](https://www.virginiabalseiro.com/) for building [Women of Solid](https://www.womenofsolid.org/) and [Digita](https://www.digita.ai/) for their [introduction videos](https://www.youtube.com/channel/UCA22hu-0VEHt5tCc7jad74g) as well as their contributions in discovering and reporting bugs.
```

Next, Ruben Verborgh gave updates and a demo of the Community Solid Server (CSS). Recently, CSS’s version 0.8 was released with 1.0 on the horizon. He also created a [chat from his pod](https://css.verborgh.org/public/2021/Solid%20World/Chat/), where attendees were able to engage with during the event. 

For Inrupt’s update, Osmar Olivo, VP of Product, gave us a sneak peek that  Enterprise Solid Server (ESS) version 1.1 will be released in early spring. He also reminded us to try out and [give feedback](https://inrupt.atlassian.net/servicedesk/customer/portal/7) for [Pod Spaces](https://signup.pod.inrupt.com/).  

> Next were our two guest presenters for Solid World. We heard from Dmitri Zagidulin and Liam Broza from[LifeScope](https://lifescope.io/), who let us know about their work with Solid. They are working on creating a “companion intelligence” by using Artificial Intelligence (AI) to create a digital memory of yourself, one that builds up a more accurate, personalized, and improved experience on the web. They will be using Solid to help with decentralizing their data storage and authorization. You can access their slides [here](https://docs.google.com/presentation/d/1vg9y_o0KtaC1j9w6yxpukEeoOsdbMCQa2zh1hQO2pYw).  

Reviewer: Missing a space here:

```suggestion
Next were our two guest presenters for Solid World. We heard from Dmitri Zagidulin and Liam Broza from [LifeScope](https://lifescope.io/), who let us know about their work with Solid. They are working on creating a “companion intelligence” by using Artificial Intelligence (AI) to create a digital memory of yourself, one that builds up a more accurate, personalized, and improved experience on the web. They will be using Solid to help with decentralizing their data storage and authorization. You can access their slides [here](https://docs.google.com/presentation/d/1vg9y_o0KtaC1j9w6yxpukEeoOsdbMCQa2zh1hQO2pYw).  
```

> Last but not least, we heard from Dick Van Gelder from Creafree. He discussed the importance of us coming together as a collective to support and protect intellectual property. Working with existing laws, we can use Solid to build a space for people to create and share their creations. The ability to gather and use all your data with different applications is just a start. You can access his slides [here](https://ln2.sync.com/dl/53b6274d0/fcs466su-bs8bgykx-hkfmpz2a-rmwamczt/view/default/9895814900003). 

Reviewer: I am assuming that he's Dutch, in which case "van" is not capitalised (woohoo, my being Dutch is relevant!) :)

```suggestion
Last but not least, we heard from Dick van Gelder from Creafree. He discussed the importance of us coming together as a collective to support and protect intellectual property. Working with existing laws, we can use Solid to build a space for people to create and share their creations. The ability to gather and use all your data with different applications is just a start. You can access his slides [here](https://ln2.sync.com/dl/53b6274d0/fcs466su-bs8bgykx-hkfmpz2a-rmwamczt/view/default/9895814900003). 
```

We continue to build steps to give individuals, organizations, and developers the ability to use data in a new, innovative, and valuable way. Solid is always growing in size and contributions. As we continue through this year, it will be exciting to see how we can grow in interoperability and integrations together!

 
### In Other News

* Solid Networking Event – March 26, 2021 at 10 AM Eastern Time

#### Community Spotlight

> * Angelo Veltens – For job testing the chat apps that were posted on the forum recently, and working with the developers to help them pinpoint issues he encountered. Also got back to someone who did not manage to get Solid working in a browser extension a while ago, sharing how he now did manage to do that.

Reviewer: Not sure if this covers the intent, but some preposition is needed here I think.

```suggestion
* Angelo Veltens – For his work testing the chat apps that were posted on the forum recently, and working with the developers to help them pinpoint issues he encountered. Also got back to someone who did not manage to get Solid working in a browser extension a while ago, sharing how he now did manage to do that.
```
